ibis/backends/base/sql/alchemy/registry.py

```diff
@@ -161,20 +161,23 @@ def _not_contains(t, expr):
 def reduction(sa_func):
     def formatter(t, expr):
         op = expr.op()
-        *args, where = op.args
-
-        return _reduction_format(t, sa_func, where, *args)
+        if op.where is not None:
+            arg = t.translate(op.where.ifelse(op.arg, ibis.NA))
+        else:
+            arg = t.translate(op.arg)
```

Comment:
I guess you could also eliminate the duplicated call to `t.translate`, but probably not worth it :)

Reply:
Then I'm going to leave it as is.